views/postgres.ejs

```diff
@@ -1,19 +1,74 @@
 <%- include("./partials/head") %>
+<h3>Learn PostgreSQL</h3>
+<div id="introduction">
```

Comment:
could we have renamed this to `database.ejs` to be reusable for other db as well?